client/components/emails/email-step-side-bar/index.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { Icon } from '@wordpress/icons';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import Gridicon from "calypso/components/gridicon";
+
+class EmailStepSideBar extends React.Component {
```

Comment:
Given that this component is just a basic content render, it may be simpler to write it as a functional component with a single JSX block, with `useTranslate` from `i18-calypso` at the top to handle the translation.